docker/ubuntu-16.04/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM  ubuntu:16.04
```

Comment:
I'm using Debian buster for things like release management, would you like to test Docker with a Debian buster image too?